src/pages-and-resources/discussions/DiscussionsSettings.test.jsx

```diff
@@ -191,6 +191,29 @@ describe('DiscussionsSettings', () => {
     });
   });
 
+  describe('Check Stepper error message', () => {
+    beforeEach(async () => {
+      axiosMock.onGet(getAppsUrl(courseId)).reply(200, legacyApiResponse);
+      renderComponent();
+      history.push(`/course/${courseId}/pages-and-resources/discussion`);
+      await waitForElementToBeRemoved(screen.getByRole('status'));
```

Comment:
which element is this?

Reply:
this is loading component which appears when you open any form.

Reply:
Ah I see, thanks.